xknxproject/loader/project_loader.py

```diff
@@ -78,12 +78,12 @@ class _GroupAddressLoader:
     @staticmethod
     def load(group_address_element: ElementTree.Element) -> XMLGroupAddress:
         """Load GroupAddress mappings."""
-
+        project_uid = group_address_element.get("Puid")
         return XMLGroupAddress(
             name=group_address_element.get("Name", ""),
             identifier=group_address_element.get("Id", ""),
             address=group_address_element.get("Address", ""),
-            project_uid=int(group_address_element.get("Puid")),  # type: ignore[arg-type]
+            project_uid=int(project_uid) if project_uid else None,
```

Comment:
```suggestion
            project_uid=int(project_uid) if project_uid is not None else None,
```
We should probably account for 0 too.

Maybe introduce a helper function like
```py
def optional_int_attribute(attribute: str | None) -> int | None:
    if attribute is not None:
        return int(attribute)
```
or
```py
def optional_int_attribute(element: ElementTree.Element, attribute: str) -> int | None:
    if (value := element.get(attribute)) is not None:
        return int(value)
```

Reply:
`project_uid` is a string so `"0"` is true:
```python
>>> bool("0")
True
```
but I'll add a more explicit check.

Reply:
ah right, I didn't think of the string...

Reply:
Should I still change it or is it ok like this?